plugins/modules/idrac_reset.py

```diff
@@ -3,64 +3,139 @@
 
 #
 # Dell OpenManage Ansible Modules
-# Version 7.1.0
-# Copyright (C) 2018-2022 Dell Inc. or its subsidiaries. All Rights Reserved.
+# Version 9.2.0
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
 
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 #
 
-
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-DOCUMENTATION = """
+DOCUMENTATION = r'''
 ---
 module: idrac_reset
-short_description: Reset iDRAC
-version_added: "2.1.0"
+short_description: Factory reset the iDRACs
+version_added: "9.2.0"
 description:
-    - This module resets iDRAC.
-    - "iDRAC is not accessible for some time after running this module. It is recommended to wait for some time,
-    before trying to connect to iDRAC."
+  - This module is responsible to reset the iDRAC to factory default settings.
 extends_documentation_fragment:
   - dellemc.openmanage.idrac_auth_options
+options:
+  reset_to_default:
```

Comment:
Add version added in documentation of module

Reply:
Done!